tap_hubspot/__init__.py

```diff
@@ -849,7 +854,9 @@ def sync_owners(STATE, ctx):
     params = {}
     if CONFIG.get('include_inactives'):
         params['includeInactives'] = "true"
-    data = request(get_url("owners"), params).json()
+    
+    # ADDED LINE HERE TO GET RESULTS
+    data = request(get_url("owners"), params).json()['results']
```

Comment:
We probably don't need comments like this in the code and instead people can rely on `git blame` to understand from the commit message why/when this change was done :)